engine/installation/linux/docker-ce/ubuntu.md

```diff
@@ -29,6 +29,7 @@ To learn more about Docker EE, see
 To install Docker CE, you need the 64-bit version of one of these Ubuntu
 versions:
 
+- Artful 17.10 (edge only)
```

Comment:
It will be in stable as well once the next stable is there (so at that point we can remove that part)